reframe/core/shell.py

```diff
@@ -4,7 +4,7 @@
 import abc
 
 
-_RFM_TRAP_ERROR = '''
+_RFM_TRAP_ERROR = r'''
```

Comment:
Question: Which python version are you running and you're getting the string warnings?

Reply:
Pycodestyle shows me the warnings regarding invalid escape sequences. You must run with warnings enabled in order to take then from Python.